recipes/graylog-forwarder/files/post-install

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+set -e
+
+user="graylog-forwarder"
+group="graylog-forwarder"
+datadir="/usr/share/graylog-forwarder"
```

Comment:
This should rather be `/var/lib/graylog-forwarder`. And we need to set `data_dir` in the forwarder config file accordingly. Writing state into `/usr/share` doesn't adhere to packaging and file system standards.

Reply:
This is here so we can `chown` the files that get laid down to the `graylog-forwarder` user. If we don't do that, they'll all be owned by root.

Reply:
The files in `/usr/share/graylog-forwarde` don't need to be owned by the forwarder user. The forwarder is not suposed to write into that directory. These files are usually owned by root.

Reply:
Okay, I get it.

Reply:
Alright, this should be fixed.